### PR TITLE
Create @fluidframework/azure-service-utils and re-export generateToken

### PR DESCRIPTION
**Pull request by sumedhb1995**
- Create a new `@fluidframework/azure-service-utils` package for helper utilities that we expect developers to use for setting up their own backends to complement their frontend applications using the `AzureClient`
- Add `generateToken` re-export so that developers can set up an API that fulfills the requirements for the token requests from an `ITokenProvider` implementation

### CODE REVIEW

File: packages/framework/azure-service-utils/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# @fluidframework/azure-service-utils
+
+Helper utilities for use when developing BE APIs for use with applications built using the `AzureClient`
```
Comment (skylerjokiel) [inline packages/framework/azure-service-utils/README.md:3]:
```suggestion
A set of helper utilities for building backend APIs in use with Azure Fluid Relay service.
```

File: packages/framework/azure-service-utils/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export { generateToken } from "@fluidframework/server-services-client";
```
Comment (skylerjokiel) [inline packages/framework/azure-service-utils/src/index.ts:6]: Devs also need ScopeType for defining permissions.

```suggestion
export { ScopeType } from "@fluidframework/protocol-definitions";
export { generateToken } from "@fluidframework/server-services-client";
```

Comment (sumedhb1995) [reply packages/framework/azure-service-utils/src/index.ts:7]: Added this + dependency update

**Review by skylerjokiel — APPROVED** [review]
@sumedhb1995 / @tylerbutler , Let's think about how we surface this package up.

**Comment by sumedhb1995** [comment]
> @sumedhb1995 / @tylerbutler , Let's think about how we surface this package up.

Do you mean in documentation?

**Pull request merged**